packages/parser/src/Rules.js

```diff
@@ -196,6 +198,7 @@ export const ParseRules: {[name: string]: ParseRule} = {
   Type(token: Token) {
     return token.value === '[' ? 'ListType' : 'NonNullType';
   },
+  InputUnionValue: [p('{'), list('ObjectField'), p('}')],
```

Comment:
I couldn't figure out how to get `Value` to return this in the case statement above